client/homebrew/brewRenderer/brewRenderer.jsx

```diff
@@ -28,14 +27,16 @@ const INITIAL_CONTENT = dedent`
 	<base target=_blank>
 	</head><body style='overflow: hidden'><div></div></body></html>`;
 
+import { safeHTML } from './safeHTML.js';
```

Comment:
Let's move the `import` up next to the other `require` lines.

Reply:
This is relocated in the current commit.